misc/tem_helpers.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+
+from definitions import ROOT_DIR
+
+sys.path.append(os.path.join(ROOT_DIR, 'tem', 'script'))
+
+import numpy as np
+import numpy.typing as npt
+
+from tem.script.model import TopicEvolution
+from tem.script.nlp import get_structured_corpus
+from tem.script.process import TEM
+from tem.script.visualization import graph
+
+
+tem_default_params = np.array([
+    0.5, # c
+    0, # alpha
+    0, # beta
+    0, # gamma
+    1, # delta
+    0.35, # theta
+    0.35, # merge_threshold
+    0.6 # evolution_threshold
+])
```

Comment:
Idea: Maybe we also want to move them into Definitions or .env file to have parts that can/ should be modified in one place

Reply:
good point! done in 902e275